runelite-client/src/main/java/net/runelite/client/plugins/stretchedmode/StretchedModeConfig.java

```diff
@@ -55,10 +55,20 @@ default boolean increasedPerformance()
 	@ConfigItem(
 		keyName = "integerScaling",
 		name = "Integer Scaling",
-		description = "Forces use of a whole number scale factor"
+		description = "Forces use of a whole number scale factor when stretching."
 	)
 	default boolean integerScaling()
 	{
 		return false;
 	}
+
+	@ConfigItem(
+		keyName = "scalingFactor",
+		name = "Resizable Scaling (%)",
+		description = "In resizable mode, the game is reduced in size this much before it's stretched."
```

Comment:
We should probably add positions to these config items now

Reply:
Well, as the options are now alphabetically sorted, the order would be
* Increased performance mode
* Integer scaling
* Keep aspect ratio
* Resizable scaling

What seems fine, I think